docs/intro/installation.rst

```diff
@@ -60,6 +60,16 @@ DataLad and all of its software dependencies (including the Git-annex-standalone
    The installer automatically configures the shell to make conda-installed
    tools accessible, so no further configuration is necessary.
 
+   .. note::
+
+      Currently, the latest DataLad version on conda is 0.11, but the handbook
+      is written for version 0.12. If you have installed DataLad 0.11 already, an upgrade
+      to the most recent 0.12 release candidate is possible with
+
+      .. code-block:: bash
+
+         pip install datalad==0.12.0rc5
```

Comment:
If you use the `~=`, this will be more resilient to future RCs and transparently shift to 0.12.x once you start having final releases.

```suggestion
         pip install datalad~=0.12.0rc5
```

This will install the newest of `0.12.0rc5`, `0.12.0rc6`, `0.12.1`, `0.12.2`, etc., but not `0.12.1rc1`, `0.13.0rc1` or `0.13.0`. If you want to catch `0.12.1rc1`, you should add the `--pre` flag.

Reply:
Thanks so much, @effigies!

Reply:
That is great. Didnt know... thx